x-pack/plugins/fleet/server/types/models/policy_preconfig.ts

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import { i18n } from '@kbn/i18n';
+import { schema } from '@kbn/config-schema';
+
+const semverRegex = () =>
```

Comment:
I think we use the library `semver` in a few places in Fleet already, might that library have a validation helper we can leverage instead of a regex test?